File: example/bertcli.rb
require 'bertrpc'

##
# Simple client code in ruby that calls our aberth service
##

svc = BERTRPC::Service.new('localhost', 10001)
r = svc.call.example.adder(1234, 4321)
puts r
r = svc.call.dictoid.to_dict("aberth", "is", "great!")
puts r

File: example/client-python/bertcli.py
#!/usr/bin/env python

"""
    Simple client in Python calling example aberth service
"""

import bertrpc
service = bertrpc.Service('localhost', 10001)
response = service.request('call').example.adder(203,302)
assert response == 505
response = service.request('call').dictoid.to_dict("aberth", "is", "great")
print response
